<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MqttClient</class>
 <widget class="QMainWindow" name="MqttClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1309</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="LinkButton">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>120</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PORTEdit">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPEdit">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Repayload">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>881</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>30</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ip</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SubEdit">
    <property name="geometry">
     <rect>
      <x>972</x>
      <y>171</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>170</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>订阅主题</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>230</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>发布：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ClientidEdit">
    <property name="geometry">
     <rect>
      <x>972</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>clientid</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserEdit">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>80</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>user</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordEdit">
    <property name="geometry">
     <rect>
      <x>972</x>
      <y>120</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SubButton">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>170</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>sub</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PubEdit">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>260</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Publpayload">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>300</y>
      <width>351</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>260</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>topic</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>430</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>payload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>260</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>pub</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>260</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Qos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="msgdidEdit">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>260</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>260</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QComboBox" name="qoscbox">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>260</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="qossubcbox">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>170</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>170</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Qos</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1309</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
